### DISCUSSION TITLE
Ran setup.sh but forgot to fix domainname: example.com in the docker-compose.yml

### DISCUSSION THREAD

**Discussion by shodanx2** [docker-mailserver/docker-mailserver#2745] [Q&A]
Although when I added the first user, I specificed the correct domain name, and the dkim config produced files in the name of my correct domain name. 

Are there places I will need to replace some example.com with my real domain  ? 

Maybe it's easier to wipe and start over !?

**Comment by casperklein** [comment]
It's all fine. Just make sure to recreate the container after your change: `docker-compose down; docker-compose up -d --force-recreate`
